### ISSUE TITLE
Copyright symbol © is identified as an emoji

### ISSUE THREAD

**Issue by smably** [jsx-eslint/eslint-plugin-jsx-a11y#388]
I have a component that renders the copyright symbol, more or less like this:
```js
render() {
  return <span className={CSS.copyright}>© 2018</span>;
}
```

This gets flagged by `jsx-a11y/accessible-emoji`.

I'm not an expert on this subject, but based on a quick Google, "©" should be read out correctly by all screen readers (e.g., https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/, see "safe characters" section), so this seems to me like a false positive.

**Comment by evcohen** [comment]
ah, good call! we can ignore these safe characters by default & I don't think this needs to be configurable. thanks for reporting.

**Comment by martynhoyer** [comment]
Just got this myself with the encoded version, as per:

```
<p>
  &copy; {new Date().getFullYear()} {fullCompanyName}
</p>
```

**Comment by qw-in** [comment]
This has been fixed as of `emoji-regex@6.5.1`.

It does however bring up another issue based on the article linked above. For instance `&sect;` is no longer caught under the `accessible-emoji` rule but perhaps should be considering "...*at least one screen reader will not read the character out loud*"[<sup>1</sup>](https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/)

**Comment by ari-os310** [comment]
Now that we're on [emoji-regex@9.0.0 ](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/package.json#L68), is there anything else to do with this issue?

**Comment by ljharb** [comment]
Seems like 29d20f78b6437511bed6524d5b1faf5ea9ad9a03 should have fixed this, indeed.
